expandable-section/expandable-section.js

```diff
@@ -11,6 +11,37 @@ export default Component.extend({
 		height: { type: "number", default: 300 },
 		title: { type: "string", default: "Expandable Section" },
 		collapsible: { type: "boolean", default: true },
+		/**
+		 * The section element
+		 */
+		sectionEl: {
+			type: "any"
+		},
+		/**
+		 * Section header
+		 */
+		sectionTitle: {
+			type: "any"
+		},
+		/**
+		 * the actual section height (height property is for max-height)
+		 */
+		sectionHeight: {
+			value({lastSet, listenTo, resolve}) {
+				resolve(lastSet.get());
+				listenTo("sectionEl", (ev, el) => {
```

Comment:
Could this just be a getter? Also, can you write tests for these?

Reply:
I wrote tests and getters.

I'm not able to reproduce the double scrollbar, when you are online, can you give me more details to reproduce it?